internal/cli/main.go

```diff
@@ -227,6 +227,16 @@ Database drivers: `+strings.Join(database.List(), ", ")+"\n")
 		}
 
 	case "drop":
+		log.Println("Are you sure you want to drop the entire database? [y/N]")
+		var response string
+		fmt.Scanln(&response)
+		response = strings.ToLower(strings.TrimSpace(response))
+
+		if response != "y" {
+			log.fatal("Aborted dropping the entire database")
+			os.Exit(1)
```

Comment:
`os.Exit()` is unnecessary since `log.fatal()` will call `os.Exit()`

Reply:
Thanks I didn't realise there were side effects to `log.fatal()`.